frontend/src/features/properties/map/MapView.tsx

```diff
@@ -82,63 +65,17 @@ const MapView: React.FC<MapViewProps> = (props: MapViewProps) => {
   const lotSizes = fetchLotSizes();
   const dispatch = useDispatch();
 
-  const getApiParams = (e: MapViewportChangeEvent): IPropertySearchParams | null => {
-    if (!e || !e.bounds) {
-      return null;
-    }
-    const {
-      pid,
-      address,
-      administrativeArea,
-      projectNumber,
-      agencies,
-      classificationId,
-      minLotSize,
-      maxLotSize,
-      inSurplusPropertyProgram,
-      inEnhancedReferralProcess,
-    } = e.filter ?? {};
-
-    const ne = e.bounds.getNorthEast();
-    const sw = e.bounds.getSouthWest();
-    const apiParams: IPropertySearchParams = {
-      pid: pid ?? null,
-      neLatitude: ne.lat,
-      neLongitude: ne.lng,
-      swLatitude: sw.lat,
-      swLongitude: sw.lng,
-      address: address ?? null,
-      administrativeArea: administrativeArea ?? null,
-      projectNumber: projectNumber ?? null,
-      agencies: agencies ?? null,
-      classificationId: classificationId ?? null,
-      minLandArea: minLotSize ?? null,
-      maxLandArea: maxLotSize ?? null,
-      inSurplusPropertyProgram: inSurplusPropertyProgram ?? null,
-      inEnhancedReferralProcess,
-    };
-    return apiParams;
-  };
   const saveLatLng = (e: LeafletMouseEvent) => {
     if (!props.disabled) {
       dispatch(saveLeafletMouseEvent(e));
     }
   };
 
-  const throttledFetch = useRef(
```

Comment:
`InventoryLayer` now handles the request for properties.  Not sure if I need to apply throttling to that layer as testing so far appears to work well without it.